src/main/kotlin/com/kotlin/boardproject/domain/post/dto/MyScrapPostResponseDto.kt

```diff
@@ -1,7 +1,7 @@
-package com.kotlin.boardproject.dto.post
+package com.kotlin.boardproject.domain.post.dto
 
-import com.kotlin.boardproject.common.enums.PostStatus
-import com.kotlin.boardproject.model.ScrapPost
+import com.kotlin.boardproject.global.enums.PostStatus
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end